[2025-08-07 11:11:53,240] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-07T11:09:40.311429+00:00 [queued]>
[2025-08-07 11:11:53,266] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-07T11:09:40.311429+00:00 [queued]>
[2025-08-07 11:11:53,267] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 11:11:53,267] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-07 11:11:53,268] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 11:11:53,298] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): verify_pipeline> on 2025-08-07 11:09:40.311429+00:00
[2025-08-07 11:11:53,317] {standard_task_runner.py:52} INFO - Started process 526 to run task
[2025-08-07 11:11:53,320] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'verify_pipeline', 'scheduled__2025-08-07T11:09:40.311429+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmp3eeyg5x6', '--error-file', '/tmp/tmp895017eh']
[2025-08-07 11:11:53,323] {standard_task_runner.py:80} INFO - Job 40: Subtask verify_pipeline
[2025-08-07 11:11:53,453] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-07T11:09:40.311429+00:00 [running]> on host b22dcc1493a4
[2025-08-07 11:11:53,906] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alla
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=verify_pipeline
AIRFLOW_CTX_EXECUTION_DATE=2025-08-07T11:09:40.311429+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-07T11:09:40.311429+00:00
[2025-08-07 11:11:53,907] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-07 11:11:53,909] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            echo " Attente traitement Logstash..." && sleep 30 &&\n            curl -s "http://elasticsearch:9200/_cat/indices/weather*" || echo "Pas encore d\'index weather"\n        ']
[2025-08-07 11:11:53,933] {subprocess.py:85} INFO - Output:
[2025-08-07 11:11:53,937] {subprocess.py:92} INFO -  Attente traitement Logstash...
[2025-08-07 11:12:24,008] {subprocess.py:92} INFO - yellow open weather_data wCNHBp2xSKuPJ9di62P4DQ 1 1 1 0 5.6kb 5.6kb
[2025-08-07 11:12:24,012] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-07 11:12:24,078] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=verify_pipeline, execution_date=20250807T110940, start_date=20250807T111153, end_date=20250807T111224
[2025-08-07 11:12:24,121] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-07 11:12:24,187] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
